src/stactools/cop_dem/stac.py

```diff
@@ -23,7 +23,7 @@
 from pystac.media_type import MediaType
 
 import rasterio
-from shapely.geometry import mapping, box
+from shapely.geometry import mapping, box, shape as make_shape
```

Comment:
Not necessary here, but this why I generally prefer to not directly import functions, since they so often can conflict w/ variable names -- `import shapely.geometry` might be verbose but its at least explicit :-P